### PR TITLE
enable Novnc

### PR DESCRIPTION
**Pull request by Fuseteam**
this is recreation of #57 with the requested changes and without the fluff
i recreated the pr because the downstream of the old pr only has one branch, the master branch. and i didn't want to have a back and forth with them creating a new branch with just the needed changes

also I didn't want to open a issue for this so I'll just ask here:
you noticed you decided to deprecate aind in favor of redroid but redroid doesn't suit my use case yet (for one I cannot get it up and running as opposed to aind and I am unable to get sensible debug info from it) would you be willing to still merge the occasional pull request for aind?

### CODE REVIEW

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
 # test
     - run: ./hack/translate-dockerfile-runopt-directive.sh < Dockerfile | DOCKER_BUILDKIT=1 docker build -f - -t aind/aind:local .
     - run: sudo ./hack/install-kmod.sh
-    - run: docker run -td --name aind --privileged -p 5900:5900 -v /lib/modules:/lib/modules:ro aind/aind:local
+    - run: docker run -td --name aind --privileged -p 5900:5900 -p 8080:8080 -e "NOVNC=1" -v /lib/modules:/lib/modules:ro aind/aind:local
```
Comment (AkihiroSuda) [inline .github/workflows/main.yaml]: s/NOVNC/WEBMODE/

File: README.md [hunk 2]
```diff
@@ -63,6 +63,10 @@ sudo modprobe binder_linux
 
 ```bash
 docker run -td --name aind --privileged -p 5900:5900 -v /lib/modules:/lib/modules:ro aind/aind
+```
+
+```bash
+docker run -td --name aind --privileged -p 8080:8080 -e "NOVNC=1" -v /lib/modules:/lib/modules:ro aind/aind
```
Comment (AkihiroSuda) [inline README.md]: s/NOVNC/WEBMODE/, and needs explanation

File: README.md [hunk 3]
```diff
@@ -158,7 +162,7 @@ aind is expected to be used in conjunction with encryption of the client device,
 
 ## TODOs
 * Map different UID range per containers
-* Support remote connection from phones and tablets, ideally using Web browsers (noVNC?).
+* Support remote connection from phones and tablets, ideally using Web browsers (noVNC support).
```
Comment (AkihiroSuda) [inline README.md]: No longer TODO?

File: docker-compose.yml [hunk 4]
```diff
@@ -4,7 +4,10 @@ services:
       image: 'aind/aind'
       privileged: true
       tty: true
+      environment:
+        - WEBMODE=0
       ports:
         - '5900:5900'
+        - '8080:8080'
```
Comment (AkihiroSuda) [inline docker-compose.yml:11]: Needs WEBMODE=1

File: README.md (hunk 2 above)
Comment (Fuseteam) [reply README.md]: like this maybe?

Comment (Fuseteam) [reply README.md]: @AkihiroSuda any thoughts on the wording in the read me?

**Review by AkihiroSuda — APPROVED** [review]
Thanks, I'll clean up the document in a follow-up commit: https://github.com/aind-containers/aind/pull/75

**Pull request merged**